backend/src/core/config.ts

```diff
@@ -362,14 +362,20 @@ const config = convict({
           pre: [],
           a: ['href'],
         },
-        safeAttrValue: (
+        onTagAttr: (
           tag: string,
           name: string,
-          value: string
+          value: string,
+          isWhiteAttr: boolean
         ): string | void => {
           // Handle Telegram mention as xss-js does not recognize it as a valid url.
-          if (tag === 'a' && name === 'href' && value.startsWith('tg://')) {
-            return value
+          if (
+            isWhiteAttr &&
+            tag === 'a' &&
+            name === 'href' &&
+            value.startsWith('tg://')
+          ) {
+            return `href="${value}"`
           }
```

Comment:
```suggestion
          }
          return xss.safeAttrValue(tag, name, value, xss.cssFilter)
```

I actually agree that `safeAttrValue` is more suitable for this use case since we're modifying the value and not the attribute itself. 

The package [exports the default `safeAttrValue`](https://github.com/leizongmin/js-xss/blob/1e3bda7871635bf7c35b7ee4cb5c84842cc14e2c/lib/index.js#L26), so I thought we could stick with the original implementation and add a fallback on the default. But this effectively breaks our ability to pass in customized `cssfilter` options - we don't use it but not sure if we're ok with that?

Reply:
Yeah that definitely looks cleaner. I think it's unlikely we will need to use a customized `cssfilter` for Telegram. Will change to use `safeAttrValue` and merge in. Thanks!